src/pages/ScholarshipForm.jsx

```diff
@@ -20,20 +20,15 @@ function ScholarshipForm() {
     return (e) =>
       setFormFieldStates({ ...formFieldStates, [id]: e.target.value });
   }
-  function updateMinMax(value) {
-    setFormFieldStates({
-      ...formFieldStates,
-      minAmount: value,
-      maxAmount: value,
-    });
-  }
-  function updateAmount(id) {
-    return (e) =>
-      setFormFieldStates({
-        ...formFieldStates,
-        [id]: parseInt(e.target.value, 10) || '',
-      });
+
+  function updateAmount(id, value, id2 = '') {
+    if (id2) {
+      setFormFieldStates({ ...formFieldStates, [id]: value, [id2]: value });
+    } else {
+      setFormFieldStates({ ...formFieldStates, [id]: value });
+    }
```

Comment:
I was thinking more like 
```js
function updateAmount(minAmount, maxAmount) {
  setFormFieldStates({...formFieldStates, minAmount, maxAmount});
}
```
And not worrying about which text field is being set.

Reply:
Notice I named the parameters `minAmount` and `maxAmount` explicitly for the sake of using the [object property value shorthand](https://javascript.info/object#property-value-shorthand)

Reply:
Ahh I think I got it. It looks much better for sure.